owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -1110,7 +1120,11 @@ public void setBackgroundText() {
             int message = R.string.file_list_loading;
             if (!mSyncInProgress) {
                 // In case file list is empty
-                message = R.string.file_list_empty;
+                if(mOnlyAvailableOffline){
```

Comment:
You could do the same in just one line by using ternary operator: 

`message = mOnlyAvailableOffline ? R.string.file_list_empty_available_offline : R.string.file_list_empty;`